plugins/ffmpeg_plugin/README.md

```diff
@@ -2,14 +2,16 @@
 
 ## How To build:
 
-./build_ffmpeg_plugin.sh
+sudo ./build_ffmpeg_plugin.sh
```

Comment:
do we really need sudo? I see build_ffmpeg_plugin.sh use sudo for the privilege operation already.